core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_autocomplete.scss

```diff
@@ -13,7 +13,13 @@
 
     .p-autocomplete-input {
         all: unset;
+        border: none;
+        font-family: $font-default;
+        font-size: $font-size-md;
         color: $black;
+        padding: 0;
+        margin: 0;
+        height: 2.3125rem; // This input height is to match the 40px height, 37px + 3px of border
```

Comment:
Var?

Reply:
I can do a calculation instead to maintain the variables

Reply:
I think you can do `calc(input-height - border-size)`

Reply:
Done!